.gitignore

```diff
@@ -59,6 +59,8 @@ dumped_data/*
 *-coverage.xml
 *-junit-report.xml
 dm/ui/dist
+dm/ui/src/routes.tsx
```

Comment:
curious why this file is ignored, is it auto-generated?

Reply:
yeah, I wrote a vite plugin to generate this file system based routing, it exports a routes object and a pages object defined in page component:

```
import xxx from xxx;

export const routes = [
  { path: '/', element: xxx},
  //...
]

export const pages = [
   { route: 'xxxx', meta: xxx }
]
```

Or it can be exported via "virtual module" but it breaks hmr, I might change this later to have better DX.